graaljs/graaljs-spring-boot-d3/src/test/java/com/example/GraaljsSpringBootD3ApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.example;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class GraaljsSpringBootD3ApplicationTests {
```

Comment:
Rename including file name.
```suggestion
import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;

@SpringBootTest
class GraalJSSpringBootD3ApplicationTests {
```